Journalator_Display/Source/Display/InvestONator/PortfolioDisplay.lua

```
@@ -0,0 +1,291 @@
+    self.CreateButton:SetScript("OnClick", function()
+      if self.CreateDialog then
+        self.CreateDialog:Show()
+      end
+    end)
+  end
+end
+
+---Create a frame for displaying a portfolio
+---@param portfolioId number The ID of the portfolio
+---@param portfolio PortfolioData The portfolio data
+<<<<<<< HEAD
```

Luacheck (1.2.0)

[error] expected statement near '<<'

(E011)